src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -342,7 +342,7 @@ ChannelsStatusResponse getChannelsStatus(List<SnowflakeStreamingIngestChannelInt
 
       return response;
     } catch (IOException | IngestResponseException e) {
-      throw new SFException(e, ErrorCode.CHANNEL_STATUS_FAILURE);
+      throw new SFException(e, ErrorCode.CHANNEL_STATUS_FAILURE, e.getMessage());
```

Comment:
Could you do this for ErrorCode.OPEN_CHANNEL_FAILURE, ErrorCode.REGISTER_BLOB_FAILURE, ErrorCode.CLIENT_CONFIGURE_FAILURE as well? I think it's redundant to pass the message again, but if you want to add it, please do it for all APIs

Reply:
IMO, it is not redundant because it will be used to construct the message which is written in ```0024=Get channel status request failed: {0}.```

Reply:
Will add in other messages too!